### PR TITLE
docs: templates.ocf.berkeley.edu

### PR DESCRIPTION
**Pull request by chrisdliu**

### CODE REVIEW

File: ocfweb/docs/docs/staff/tips/emailtemplates.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[[!meta title="Email Templates"]]
+
+General email response templates can be found at `templates.ocf.berkeley.edu`.
+
+Some templates have variables that should be filled out before copying the template.
```
Comment (abizer) [inline ocfweb/docs/docs/staff/tips/emailtemplates.md]: The template variables will be automatically filled out by entering text into the input boxes above the main text box.

File: ocfweb/docs/docs/staff/tips/emailtemplates.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[[!meta title="Email Templates"]]
+
+General email response templates can be found at `templates.ocf.berkeley.edu`.
```
Comment (chriskuehl) [inline ocfweb/docs/docs/staff/tips/emailtemplates.md]: make this a link?

File: ocfweb/docs/docs/staff/tips/templates.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[[!meta title="Email Templates"]]
+
+General email response templates can be found at [templates.ocf.berkeley.edu](https://templates.ocf.berkeley.edu) for purposes such as sending requests for DNS to the University hostmaster and notifying banned users. Templates should be used when applicable because they are designed to provide all necessary information while remaining cordial to the recipient.
```
Comment (kpengboy) [inline ocfweb/docs/docs/staff/tips/templates.md]: Can you wrap these lines to 80 characters?

Comment (kpengboy) [reply ocfweb/docs/docs/staff/tips/templates.md]: It's rather rare for us to ban users, the templates user-how-to-get-vhost (for people who come asking "how do I get mygroup.berkeley.edu") and user-alumni-reset-pass (for alumni who can't reset their passwords online) are more commonly used examples of templates

**Comment by kpengboy** [comment]
@chrisdliu looks good, feel free to merge it any time

**Pull request merged**